datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -288,6 +287,25 @@ def create_scraper_configuration(self, instance=None):
         if config['metadata_metric_name'] and config['metadata_label_map']:
             config['_default_metric_transformers'][config['metadata_metric_name']] = self.transform_metadata
 
+        # Set up the HTTP wrapper for this endpoint
```

Comment:
I would suggest moving this whole section to a specific functions e.g. `setup_http_handlers()`.

`create_scraper_configuration` is already too big :)